packages/frontend/src/fetcher/abstraction/index.ts

```diff
@@ -7,20 +7,20 @@ export interface SupportedInChainParams {
 }
 
 export interface FetchQuestionParams {
-    provider: Provider;
+    nodeClient: PublicClient;
```

Comment:
Here we'd ideally want to keep it consistent with the SDK. The name I've used is `publicClient`. The name `nodeClient` I've used in the CCT repo to indicate the client was for the local node running on the user's machine.

Reply:
Fixed in a4ffc7267d051f0b931b2e398f9889d23fe263df.